### PR TITLE
Fix "what SYMBOL" issue 

### PR DESCRIPTION
**Pull request by sterrettJD**
`parse_ko()` has been raising an error for my use (through AMON, consistent with [AMON issue #13](https://github.com/lozuponelab/AMON/issues/13). When running `test_parse_KEGG.py`, tests for this function fail.

A typical error message would be `ValueError: What is SYMBOL in K00001?` from `raise ValueError('What is {} in {}?'.format(current_entry_name, ko_dict['ENTRY']))` at [line 212](https://github.com/lozuponelab/KEGG_parser/blob/cbef7b23d3a68cd79930557dcaf392c88664c6ce/KEGG_parser/parsers.py#L212). 

To circumvent this issue, I added SYMBOL to `NOT_CAPTURED_KO_FIELDS`. But then I got a similar issue at each of the next steps with the CO fields and PATHWAY fields, so I added NETWORK and INCLUDING to those (respectively). After these changes, all tests pass, and AMON is no longer erroring.

I could be wrong, but I'm guessing these fields have been added to the KEGG reports since KEGG_Parser was last updated? Any ideas on if this is the right approach to fixing this issue?

### CODE REVIEW

File: KEGG_parser/parsers.py [hunk 1]
```diff
@@ -194,15 +194,16 @@ def split_module_reaction(current_dict, current_entry_name, current_entry_data):
...
 NOT_CAPTURED_RN_FIELDS = ('REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL')
 
 NOT_CAPTURED_CO_FIELDS = ('BRITE', 'ATOM', 'BOND', 'BRACKET', 'ORIGINAL', 'REPEAT', 'NODE', 'EDGE', 'SEQUENCE',
                           'GENE', 'ORGANISM', 'TYPE', 'EFFICACY', 'PRODUCT', 'CLASS', 'DISEASE', 'TARGET',
-                          'METABOLISM', 'INTERACTION', 'STR_MAP', 'REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL')
+                          'METABOLISM', 'INTERACTION', 'STR_MAP', 'REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL',
+                          'NETWORK')
 
-NOT_CAPTURED_PATHWAY_FIELDS = ('GENE', 'ORGANISM', 'REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL')
+NOT_CAPTURED_PATHWAY_FIELDS = ('GENE', 'ORGANISM', 'REFERENCE', 'AUTHORS', 'TITLE', 'JOURNAL', 'INCLUDING')
```
Comment (kthurimella) [inline KEGG_parser/parsers.py:206]: This makes sense and it looks like it must have been added afterwards. I'm wondering if there is a case where we can capture fields that may get added, but we don't update the not captured fields, just to keep a bit less of a dependency on keeping up with KEGG fields.

**Review by kthurimella — APPROVED** [review]
LGTM otherwise. Not sure where else we can make improvements. Thanks for adding these fields in.

File: KEGG_parser/parsers.py (hunk 1 above)
Comment (sterrettJD) [reply KEGG_parser/parsers.py:206]: I agree. Hopefully we can discuss ways to do this during our next code review! @JTFouquier @bsantan @acolorado1 @albatalavera @jboconnor13 (tagging you all so we can discuss at some point)

Comment (sterrettJD) [reply KEGG_parser/parsers.py:206]: Didn't have a chance to specifically discuss a better fix, but did decide to go ahead and merge.

**Pull request merged**